.circleci/local_publish_helpers.sh

```diff
@@ -296,9 +296,19 @@ function runE2eTestCb {
     if [ -f  $FAILED_TEST_REGEX_FILE ]; then
         # read the content of failed tests
         failedTests=$(<$FAILED_TEST_REGEX_FILE)
-        NODE_V8_COVERAGE=$E2E_TEST_COVERAGE_DIR yarn e2e --forceExit --no-cache --maxWorkers=4 $TEST_SUITE -t "$failedTests"
+        if [[ ! -z "$DISABLE_COVERAGE" ]]; then
+            echo Running WITHOUT coverage
+            yarn e2e --forceExit --no-cache --maxWorkers=4 $TEST_SUITE -t "$failedTests"
+        else
+            NODE_V8_COVERAGE=$E2E_TEST_COVERAGE_DIR yarn e2e --forceExit --no-cache --maxWorkers=4 $TEST_SUITE -t "$failedTests"
+        fi
     else
-        NODE_V8_COVERAGE=$E2E_TEST_COVERAGE_DIR yarn e2e --forceExit --no-cache --maxWorkers=4 $TEST_SUITE
+        if [[ ! -z "$DISABLE_COVERAGE" ]]; then
+            echo Running WITHOUT coverage
+            yarn e2e --forceExit --no-cache --maxWorkers=4 $TEST_SUITE
+        else
+            NODE_V8_COVERAGE=$E2E_TEST_COVERAGE_DIR yarn e2e --forceExit --no-cache --maxWorkers=4 $TEST_SUITE
+        fi
```

Comment:
could we `export NODE_V8_COVERAGE=$E2E_TEST_COVERAGE_DIR` instead and refactor this if(disable coverage) outside to avoid this conditional Christmas tree

Reply:
That could work; I image we'd need to set, and unset if we are exporting, as we only want this specific command to be tracked for coverage - I'll create a ticket for this

Reply:
If this has to be localized then maybe not `export` but there should be a way to simplify this `if` tree here.